about/join.rst

```diff
@@ -0,0 +1,78 @@
+Usage model and joining
+=======================
+
+Aalto Scientific Computing operates with a community stakeholder model and is
+administered by the School of Science.  Schools, departments, and
+other units join an contribute resources, and get a fair-share of the
+output.  There are two different components to join:
+
+- HPC: Science-IT.  Get a share of computing resources via the :doc:`Triton
+  computing cluster </triton/index>`.
+- Aalto Research Software Engineers (RSE): Support of the RSE program
+  provides intensive hands-on support and service for research
+  software development.
+
+
+For everyone
+------------
+
+Aalto Scientific Computing gets university-level support already, so
+our computing resources are usable by anyone (with a limited share).
+By joining further, a unit gets something even more valuable: time.
+Our support for using our infrastructure is concentrated for member
+departments which provide joint staff with us or support the RSE
+program.
```

Comment:
departments which provide joint staff with us or support the [RSE
program](https://scicomp.aalto.fi/rse/).